comp/remote-config/rcclient/rcclientimpl/rcclient.go

```diff
@@ -311,14 +317,26 @@ func (rc rcClient) agentConfigUpdateCallback(updates map[string]state.RawConfig,
 		// Need to update the log level even if the level stays the same because we need to update the source
 		// Might be possible to add a check in deeper functions to avoid unnecessary work
 		pkglog.Infof("Changing log level to '%s' through remote config (new source)", mergedConfig.LogLevel)
-		err = rc.settingsComponent.SetRuntimeSetting("log_level", mergedConfig.LogLevel, model.SourceRC)
+		if err := rc.settingsComponent.SetRuntimeSetting("log_level", mergedConfig.LogLevel, model.SourceRC); err != nil {
+			errs = multierror.Append(errs, err)
+		}
+	}
+
+	streamLogsSource := pkgconfigsetup.Datadog().GetSource("logs_config.streaming.enable_streamlogs")
```

Comment:
With the new flareargs functionality we don't need to rely on enabling stream logs via a config entry and can instead directly be passed a TaskArg in flare.go:onAgentTaskEvent